tests/test_geometry.py

```diff
@@ -0,0 +1,7 @@
+import pytest
```

Comment:
Unfortunatly there is already a test_geometry file, which is for graphical unit tests of geometrey module. Could you please rename this file into something like "test_unit_geometry.py'" ?

Reply:
Done